groups/application/sg.tf

```diff
@@ -40,4 +39,28 @@ resource "aws_security_group_rule" "weblogic" {
   to_port           = 58032
   protocol          = "tcp"
   cidr_blocks       = var.weblogic_client_ips
+}
+
+resource "aws_security_group_rule" "logstash" {
+
+  security_group_id = module.gfn_app_ec2_security_group.this_security_group_id
+  description       = "Allow on-premise logstash traffic"
+  for_each          = toset(["22", "443"])
+  type              = "ingress"
+  from_port         = each.value
+  to_port           = each.value
+  protocol          = "tcp"
+  cidr_blocks       = var.logstash_client_ips
```

Comment:
This rule and the next share ports, you should be able to concatenate the 2 lists into a single list.